Source/include.cmake

```diff
@@ -0,0 +1,3 @@
+set(FREERTOS_CONFIG_FILE_DIRECTORY ${FW3_APPLICATION_DIR}/Include)
+
+set(FREERTOS_HEAP ${FW3_COMPONENTS_DIR}/Source/no_heap.c)
```

Comment:
When I try to build the project locally, it fails because these variables are not set. It turns out that we have to define them in the following ways:
```cmake
set(FREERTOS_CONFIG_FILE_DIRECTORY ${FW3_APPLICATION_DIR}/Include CACHE STRING "")
set(FREERTOS_PORT GCC_ARM_CM4F CACHE STRING "")
set(FREERTOS_HEAP ${FW3_COMPONENTS_DIR}/Source/no_heap.c CACHE STRING "")
```